### ISSUE TITLE
Implement temperature dependant load profile agent

### ISSUE THREAD

**Issue by t-ober** [ie3-institute/simona#248]

**Comment by staudtMarius** [comment]
Since we moved the source/implementation of profiles to the data model and only load them in SIMONA based on the given load profiles, I think it makes sense, to move this issue to the data model.

@sebastian-peter, @danielfeismann What do you think?

**Comment by sebastian-peter** [comment]
Yes, would think so. Or, we just close this one and leave it to someone who needs/wants this to open a new issue? I don't know if there's currently anyone like this on the horizon.

**Comment by danielfeismann** [comment]
Let's close this :)
